src/solver/text.h

```diff
@@ -29,7 +29,7 @@
 #define FMT06 "\n o  Retrieving project data"
 #define FMT07 "\n o  Writing output report"
 #define FMT08 \
-  "\n  EPA STORM WATER MANAGEMENT MODEL - VERSION 5.1 (Build 5.1.013)"         //(5.1.013)
+  "\n  EPA STORM WATER MANAGEMENT MODEL - VERSION 5.1 (Build 5.1.014)"         //(5.1.014)
 #define FMT09 \
```

Comment:
Would be nice to eliminate this. Easy to forget to do this. Should update version number in one place.

Reply:
Let me know what you think about my solution.

Reply:
I like this. One less place to update version number. :P